scripts/plugin-tests.js

```diff
@@ -2537,10 +2557,18 @@ let syncTests = {
         },
       ],
     })
+    const outputs = Object.keys(metafile.outputs).map(outputPath => {
+      const absolutePath = path.resolve(outputPath)
+      const relativePath = path.relative(testDir, absolutePath)
+
+      return relativePath
+    })
     const bundle = require(output)
     const result = await bundle()
 
     assert.deepStrictEqual(result.default, shimData)
+    assert(outputs.includes('out.js'))
+    assert(outputs.some(outputPath => /in-([\w\d]+)-(\d)+\.js/.test(outputPath)))
```

Comment:
[sand] The code in those functions are duplicated, should it be grouped to a single helper function?

Reply:
I'm trying to keep the changes as contained as possible, including in the tests. So for now I'd rather have some duplication than introducing or modifying more files.